queries/dart/highlights.scm

```diff
@@ -83,22 +91,44 @@
 
 ; Variables
 
+; var keyword
+(inferred_type) @keyword
+
+(const_builtin) @constant.builtin
+(final_builtin) @constant.builtin
+
 ((identifier) @type
  (#match? @type "^[A-Z]"))
 
-(this) @constant.builtin
+; TODO: currently not working
+(unconditional_assignable_selector
```

Comment:
@theHamsta I could use some help with these. These queries work in the `web-ui` but not in `nvim` there is no error or anything, just no highlighting for the fields, I'm not sure why. I'm trying to highlight methods and instance variables e.g. `thing.var` or `thing.method()` (just the `var` and `method` in those cases)

Reply:
Do you have a dart example file we can test this with?

Reply:
@stsewd here is a fake class with some assignments and variable access
```dart

class Thing {
 final int thing_id = 1;
 int other_thing_id = 2;
 Child child;
 
 updateId(int n) {
 	thing_id = n;
    this.other_thing_id = n;
    child?.value = n;
 }
}

```
I'm trying to highlight `value` and `other_thing_id`

Reply:
So, looks like there isn't a group named `identifier` defined, closest to that is `parameter` https://github.com/nvim-treesitter/nvim-treesitter/blob/6133257ed6dcaec35feaa31f9afa0245668dfe48/lua/nvim-treesitter/highlight.lua#L37

I think makes sense adding that new group

Reply:
All supported groups are (or should be) in https://github.com/nvim-treesitter/nvim-treesitter/blob/master/CONTRIBUTING.md#highlights (but the file above is the source of truth anyway)

Reply:
🤦🏿‍♂️ you're absolutely right I just saw it was used in some other `highlight.scm` files and assumed it had a highlight group